### ISSUE TITLE
Generating test dataset on Mac arm64 build fails with "No module named 'spikeinterface'"

### ISSUE THREAD

**Issue by rly** [NeurodataWithoutBorders/nwb-guide#840]
### Describe the issue

```
index-178c0fa4.js:7278 [main-process]: [2024-06-07 09:55:16,250] ERROR in app: Exception on /data/generate [POST]
Traceback (most recent call last):
  File "flask/app.py", line 1484, in full_dispatch_request
  File "flask/app.py", line 1469, in dispatch_request
  File "flask_restx/api.py", line 404, in wrapper
  File "flask/views.py", line 109, in view
  File "flask_restx/resource.py", line 46, in dispatch_request
  File "namespaces/data.py", line 19, in post
  File "manageNeuroconv/manage_neuroconv.py", line 1637, in generate_test_data
ModuleNotFoundError: No module named 'spikeinterface'
```

### Steps to Reproduce

Install 1.0 test release

### Operating System

Mac OS with Mac M1

### GUIDE Version

1.0 test

### Code of Conduct

Yes

### Did you confirm this issue was not already reported?

Yes

**Comment by rly** [comment]
Strangely a local build works on my system.

I will do some testing. Might just require adding
```
tmp_ret = collect_all('spikeinterface')
datas += tmp_ret[0]; binaries += tmp_ret[1]; hiddenimports += tmp_ret[2]
```
to `nwb-guide.spec`

**Comment by rly** [comment]
Fixed by #841
